flaky test: volume quota test of memkv

```bash
=== RUN   TestMemKVClient
2023/06/02 07:39:39.217966 juicefs[5825] <INFO>: Create session 1 OK with version: 1.1.0-dev+unknown [base.go:488]
    base_test.go:567: total space 1048576, iavail 100
2023/06/02 07:39:39.323272 juicefs[5825] <INFO>: close session 1: %!s(<nil>) [base.go:609]
--- FAIL: TestMemKVClient (0.11s)
```

refer to #3740

--- a/pkg/meta/tkv.go
+++ b/pkg/meta/tkv.go
@@ -495,7 +495,7 @@ func (m *kvMeta) flushStats() {
 				logger.Warnf("Update space stats: %s", err)
 			}
 		}
-		if inodes := atomic.SwapInt64(&m.newInodes, 0); inodes != 0 {
+		if inodes := atomic.LoadInt64(&m.newInodes); inodes != 0 {
 			if v, err := m.incrCounter(totalInodes, inodes); err == nil {
 				atomic.AddInt64(&m.newInodes, -inodes)
 				atomic.StoreInt64(&m.usedInodes, v)

